### PR TITLE
Add helptext to ansible-inventory's positional arg

### PR DESCRIPTION
**Pull request by lucanLepus**
##### SUMMARY

Just added the existing documentation as help text to make it more obvious how/when the parameter is supposed to take effect when looking at the --help output

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: lib/ansible/cli/inventory.py [hunk 1]
```diff
@@ -73,7 +73,9 @@ def init_parser(self):
         # remove unused default options
         self.parser.add_argument('--list-hosts', help=argparse.SUPPRESS, action=opt_help.UnrecognizedArgument)
 
-        self.parser.add_argument('args', metavar='host|group', nargs='?')
+        self.parser.add_argument('args', metavar='host|group', nargs='?',
+                                 help='The name of a host to match in the inventory, relevant when using --list '
```
Comment (bcoca) [inline lib/ansible/cli/inventory.py]: when using `--host`

**Comment by jpmens** [comment]
Good you added this, @lucanLepus, but I agree with @bcoca: it's with `--host`:

```console
$ cat inventory
localhost ansible_connection=local ansible_python_interpreter=/usr/bin/python3

[dbservers]
db1 port=11
db2
db3
db4 port=44

$ ansible-inventory --host db4
{
    "port": 44
}

$ ansible-inventory --list db2 -y
all:
  children:
    dbservers:
      hosts:
        db1:
          port: 11
        db2: {}
        db3: {}
        db4:
          port: 44
    ungrouped:
      hosts:
        localhost:
          ansible_connection: local
          ansible_python_interpreter: /usr/bin/python3
```

Tested with `ansible [core 2.16.0]`

**Comment by lucanLepus** [comment]
Yikes, stumbled into something here...

I agree this help text is wrong, as is https://docs.ansible.com/ansible/latest/cli/ansible-inventory.html#arguments from where i copied it without properly verifying.
The positional "host" argument seems to have no effect on `--list`.

However, I have to disagree with substituting `--host` here being correct:

`--host` also does not care about the "positional argument" this help text addresses but demands it's own argument:
```
--host HOST           Output specific host info, works as inventory script. It will ignore limit
``` 

using @jpmens inventory from above
```shell
$ ansible-inventory db1 --host
[...]
ansible-inventory: error: argument --host: expected one argument
[...]
$ ansible-inventory --host db4 -y db1
port: 44
$ ansible-inventory --host db1 -y db4
port: 11
$ ansible-inventory db4 --host db1 -y
port: 11
$ ansible-inventory db1 --host db4 -y
port: 44
$ ansible-inventory db1 --host db4 -y db2
[...]
ansible-inventory: error: unrecognized arguments: db2
[...]
```
the parser just eats up the positional argument but it has no effect when using `--host`!

It seems like the only effect this positional argument can have is as a group name when using `--graph`:
```shell
$ ansible-inventory --graph
@all:
  |--@ungrouped:
  |  |--localhost
  |--@dbservers:
  |  |--db1
  |  |--db2
  |  |--db3
  |  |--db4
$ ansible-inventory dbservers --graph
@dbservers:
  |--db1
  |--db2
  |--db3
  |--db4
```

**Comment by lucanLepus** [comment]
how about this now?
I'm positive the argument only gets used as a group in `--graph`.
Also i assume the `ARGUMENTS=` and `usage=` changes end up in the documentation!?

**Review by bcoca — APPROVED** [review]

**Review by jborean93 — APPROVED** [review]

**Pull request merged**